paasta_tools/paasta_cli/cmds/status.py

```diff
@@ -200,9 +213,20 @@ def paasta_status(args):
         cluster_filter = args.clusters.split(",")
     else:
         cluster_filter = None
+    if args.instances is not None:
+        instance_filter = args.instances.split(",")
```

Comment:
ok see this is dumb. we're still having this conversation, github. just because i pushed a commit doesn't mean we're done.